src/frontend/components/SøknadsSteg/DinLivssituasjon/useDinLivssituasjon.tsx

```diff
@@ -192,6 +209,16 @@ export const useDinLivssituasjon = (): {
         avhengighet: jobberPåBåt,
     });
 
+    const registrerteArbeidsperioder = useFelt<IArbeidsperiode[]>({
```

Comment:
Vet ikke helt hvordan vi ønsker å bruke toggelen enda, men ville bare påpeke at vi i de neste PR'ene passer på å ikke innføre en valideringsfunksjon i denne useFelt-hooken uten toggle, for da tror jeg ikke vi  får validert skjema

Reply:
Kanskje vi kunne lagt det på på skalFeltetVises f.eks. At dersom EØSerSkruddPå && .. osv

Reply:
Fjerner det vi ikke trenger for nå, og så kan vi heller se på den commiten når vi skal legge det inn igjen